i18n/de.json

```diff
@@ -1,51 +1,11 @@
 [
   {
-    "id": "April",
-    "translation": "April"
+    "id": "actiance.xml.output.formatter.marshalToXml.appError",
+    "translation": ""
   },
   {
-    "id": "August",
```

Comment:
I'm not sure why we're removing the ability to translate months for instance?

Reply:
I think it is because is no longer being used

Reply:
Aren't they used?

![image](https://user-images.githubusercontent.com/13119842/41910977-5bda1254-7919-11e8-9b84-726803e3d242.png)

Reply:
Unless those are generated differently now. But I don't see a way to translate the months to different languages,

Reply:
does are in the webapp not the server and those are generated with JS code that uses locale

Reply:
Okay I see.